src/common/utils/sort-compare.ts

```diff
@@ -53,3 +56,32 @@ export function sortCompareChartVersions(left: ChartVersion, right: ChartVersion
 
   return sortCompare(left.version, right.version);
 }
+
+
+
+export function sortCharts(charts: RawHelmChart[]) {
+  interface ExtendedHelmChart extends RawHelmChart {
+    __version: SemVer;
+  }
+
+  const chartsWithVersion = Array.from(
+    iter.map(
+      charts,
+      (chart => {
+        const __version = coerce(chart.version, { includePrerelease: true, loose: true });
```

Comment:
What is the meaning to use underscores before variable name here? `__version`?

Reply:
I meant it as both "temporary" and less likely to conflict with other fields. I chose `__version` because it is related to the `version` field but I don't want to override that field.